### PR TITLE
Change images for the rename of Organisation to Agency/Company

### PR DESCRIPTION
**Pull request by rvzakngh**

### CODE REVIEW

**Review by yilunyou — APPROVED** [review]

**Comment by yilunyou** [comment]
I've changed your base to stg. Please resolve any conflicts 
Please do not merge this until you've merged your pr and and komal has ok it for stage deployment. basically wait till ur code is merged to stage before you merge the docs

for docs, we will do 
dev > stg > master
merge to stg and master will be synchronise to each of our release trains

**Review by SenthilKumarSivakami — APPROVED** [review]
Hi @rvzakngh thank you for updating the screenshots. One small request, for consistency, going forward can we use the same style for the red arrow( to highlight CTA buttons) or please let me know so I can update the screenshots as and when needed.

Thanks.

**Review by yilunyou — APPROVED** [review]

**Pull request merged**
